.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!-- (REQUIRED) What is the nature of this issue? -->
+## This issue is a:
+[ ] Bug
+[ ] Feature suggestion
+[ ] Other
+ 
+<!-- (REQUIRED) What is the issue/current behavior? -->
+## Description:
+ 
+<!-- (OPTIONAL) What needs to be done to replicate this issue? (provide Gist if needed) -->
+### Steps to reproduce
```

Comment:
Maybe we can add "Start by describing the environment, such as OS, Magento 2 version, store setup, etc."